CMakeLists.txt

```diff
@@ -33,8 +33,6 @@ include(ProjectDefaults)
 include(Public)
 include(CXXDefaults)
 
-add_definitions(-DPXR_PYTHON_SUPPORT_ENABLED)
```

Comment:
What was the reason for removing this? @fabal mentioned something may be fixed in 0.8.2 so we can remove it. But I just wanted to double check

Reply:
This update is correct, see [here](https://github.com/PixarAnimationStudios/USD/commit/4094660a964d5f36116a18f821c40b3d7afbd53e)